cc/postgres/docker/README.md

```diff
@@ -0,0 +1,41 @@
+This image is based upon [`postgres:12`](https://hub.docker.com/_/postgres/) and adds PostgreSQL extension with several epsilon-DP aggregate functions. 
+
+# How to build this image
+
```

Comment:
I see you changed the permissions on those fils, but I'm still running into this problem.

Reply:
@dasmdasm thanks for the review. I'm not able to reproduce the problem. I've downloaded from the scratch repo and rebuilt all but unfortunately for me is ok.
Which permissions are necessary?  At the moment of writing, both files have `775` and on the container, permissions seem to be maintained.
Thanks for help

Reply:
Hmm, when I download it, the two files in question have `750`. Running `chmod 755` on them fixes the problem. I'm not sure why the permissions are showing up weird, but maybe we can add a troubleshooting note to the README in case others have the same experience?